falcon.install

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Execute code after install profile.
+ */
+function falcon_install() {
+
+  // Creating list of default custom features which should be enabled.
+  // All custom features not have dependency on 'Features' module so we should
+  // enable custom features after installed 'Features' module because
+  // it solving problem with configuration conflicts.
+  // We can add each custom features dependency on 'features' module and
+  // then we can add the custom features to falcon.info.yml.
+  $FEATURES_DEFAULT_ENABLED = [
+    'falcon_feature_appeal',
+    'falcon_feature_commerce',
+    'falcon_feature_commerce_direct_debit_gateway',
+    'falcon_feature_content_blocks',
+    'falcon_feature_donation',
+    'falcon_feature_user'
+  ];
+
+  // Enable default custom features.
+  \Drupal::service('module_installer')->install($FEATURES_DEFAULT_ENABLED);
+}
```

Comment:
It is a good practice to leave 1 blank line at the end of a file - this way you won't modify the last line with some meaninful code when adding more lines to the file - so please always have one empty line at the end of a file ;)